maintenance_plan/views/maintenance.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- maintenance.kind : views -->
+    <record id="maintenance_kind_view_tree" model="ir.ui.view">
+        <field name="name">maintenance.kind.tree</field>
+        <field name="model">maintenance.kind</field>
+        <field name="arch" type="xml">
+            <tree string="Maintenance Kinds" editable="top">
+                <field name="name"/>
+                <field name="active"/>
+            </tree>
+        </field>
+    </record>
+
+    <!-- maintenance.kind : actions -->
+    <record id="maintenance_kind_action" model="ir.actions.act_window">
+        <field name="name">Maintenance kinds</field>
+        <field name="res_model">maintenance.kind</field>
+        <field name="view_mode">tree</field>
+        <field name="view_id" ref="maintenance_kind_view_tree"/>
+    </record>
+
+    <menuitem
+        id="menu_maintenance_kind_configuration"
+        name="Maintenance kinds"
+        parent="maintenance.menu_maintenance_configuration"
+        action="maintenance_kind_action"
+        sequence="4" />
+
+
+    <!-- maintenance.plan : views -->
+    <record id="maintenance_plan_view_form" model="ir.ui.view">
+        <field name="name">maintenance.plan.form</field>
+        <field name="model">maintenance.plan</field>
+        <field name="arch" type="xml">
+            <form string="Maintenance Plan">
+                <group>
+                    <field name="maintenance_kind_id"/>
+                    <field name="period"/>
+                    <field name="duration"/>
+                    <field name="next_maintenance_date"/>
+                </group>
+            </form>
+        </field>
+    </record>
+
+    <record id="maintenance_plan_view_tree" model="ir.ui.view">
+        <field name="name">maintenance.plan.tree</field>
+        <field name="model">maintenance.plan</field>
+        <field name="arch" type="xml">
+            <tree string="Maintenance Plans">
+                <field name="maintenance_kind_id"/>
+                <field name="period"/>
+                <field name="duration"/>
+                <field name="next_maintenance_date"/>
+            </tree>
+        </field>
+    </record>
+
+    <!-- equipment : views inheritance -->
+    <record id="hr_equipment_view_form" model="ir.ui.view">
+        <field name="name">equipment.form.inherit</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form" />
+        <field name="model">maintenance.equipment</field>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='maintenance']" position="replace">
```

Comment:
Please avoid the following travis warning:

`maintenance_plan/views/maintenance.xml:62: [W7940(dangerous-view-replace-wo-priority), ]  Dangerous use of "replace" from view with priority 0 < 99
`

Instead of replacing the group, you could make it invisible and add a new group.